homeassistant/components/sensor/glances.py

```diff
@@ -142,6 +141,11 @@ def state(self):
                 return value['processcount']['thread']
             elif self.type == 'process_sleeping':
                 return value['processcount']['sleeping']
+            elif self.type == 'cpu_temp':
+                for sensor in value['sensors']:
```

Comment:
I will add a small note to the doc but I think that's outside HA scope. I had to install lm-sensors in Ubuntu for glances to start providing temperatures, but this is distro and hardware dependent. This sensor interfaces with glances, installing glances is up to the user. It doesn't either tell how to set up glances to start up on booting up, which is advisable.